linkerd/app/inbound/src/lib.rs

```diff
@@ -5,12 +5,17 @@
 
 #![deny(warnings, rust_2018_idioms)]
 
+mod allow_discovery;
+mod prevent_loop;
+mod require_identity_for_ports;
+pub mod target;
```

Comment:
nit: any reason for moving this?

Reply:
It's kind of become the idiom to put modules before imports... From a discoverability point of view: easier to see all the modules as soon as you open the file, and then imports are closer uses (and reference modules that have already been defined). Just a nit really